src/api/app/models/project_log_entry.rb

```diff
@@ -7,6 +7,15 @@ class ProjectLogEntry < ApplicationRecord
 
   validates :event_type, :datetime, :project_id, presence: true
 
+  scope :builds, -> { where(event_type: [:build_fail, :build_success]) }
+  scope :create_packages, -> { where(event_type: :create_package) }
```

Comment:
what about only one scope with a parameter for the event_types? :thinking:

Reply:
That scope is called `.where` :wink:

Reply:
mm no.... what I am proposing is:

```Ruby
scope packages_with_type, -> (type) { where(event_type: type) }
```
I just don't see the point on having that many scope for such simple conditions. I would prefer to use `where` directly than the scopes. I don't see they provide any benefit.

Reply:
I know this is what you proposing. That means:

```ruby
ProjectLogEntry.packages_with_type(:build_fail)
```
instead of

```ruby
ProjectLogEntry.where(packages_with_type :build_fail)
```

One everybody knows, one everybody has to look up at least once....

Reply:
dropped the scope commit now...